examples/custom-receiver/src/KoaReceiver.ts

```diff
@@ -0,0 +1,335 @@
+/* eslint-disable node/no-extraneous-import */
+/* eslint-disable import/no-extraneous-dependencies */
+import { InstallProvider, CallbackOptions } from '@slack/oauth';
+import { ConsoleLogger, LogLevel, Logger } from '@slack/logger';
+import Router from '@koa/router';
+import Koa from 'koa';
+import { Server, IncomingMessage, ServerResponse } from 'http';
+import {
+  App,
+  CodedError,
+  Receiver,
+  ReceiverEvent,
+  ReceiverInconsistentStateError,
+  HTTPModuleFunctions as httpFunc,
+  HTTPResponseAck,
+  InstallProviderOptions,
+  InstallURLOptions,
+  BufferedIncomingMessage,
+  ReceiverDispatchErrorHandlerArgs,
+  ReceiverProcessEventErrorHandlerArgs,
+  ReceiverUnhandledRequestHandlerArgs,
+} from '@slack/bolt';
+
+// TODO: import from @slack/oauth
```

Comment:
This issue will be resolved by https://github.com/slackapi/node-slack-sdk/pull/1446